### ISSUE TITLE
Support Conference in a Cluster with all participants on the same media server 

### ISSUE THREAD

**Issue by maria-farooq** [RestComm/Restcomm-Connect#1154]
cc: @deruelle  @leftyb 
I will add architect diagram here:

**Comment by deruelle** [comment]
@maria-farooq the title should be more like Support Conference in a Cluster with all participants on the same media server and we need #932 as a pre requisite to this issue

**Comment by maria-farooq** [comment]
link to high level diagram https://docs.google.com/drawings/d/1k-APbfIZveRtfg1J9rZ7G-lQIY7Lvqzty_qy-_4lLB4

**Comment by maria-farooq** [comment]
sequence diagram: https://www.draw.io/#G0B2CFlEFeY-zfZnNDUG9hdHh1RWs

**Comment by deruelle** [comment]
@maria-farooq I can't see the diagram apparently in draw.io. Would you mind doing it with plantuml so we can add it to the design documentation of Restcomm-Connect. 
@gvagenas shall we start a separate asciidoc guide on Restcomm Connect design  ?

**Comment by gvagenas** [comment]
closing since we implement #1104
